Makefile

```diff
@@ -62,7 +62,7 @@ build-requirements:
 	$(VIRTUALENV) $(TEMPDIR)
 	$(TEMPDIR)/bin/pip install -U pip
 	$(TEMPDIR)/bin/pip install -Ue .
-	$(TEMPDIR)/bin/pip freeze > requirements.txt
+	$(TEMPDIR)/bin/pip freeze | grep -v -- '^-e' > requirements.txt
```

Comment:
This is a pip freeze to get all the lib version that were installed at the time of the release.
The `grep` command is to not keep the git link of the dependencies (webpush-channels one in this case)